html_css/intermediate_css/browser-compatibility.md

```diff
@@ -56,8 +56,8 @@ Mobile devices mostly consist of smartphones and tablets. The most popular mobil
 
 As you're developing your applications, you must also consider whether your application should be fully mobile compatible. There are a couple of specifics about mobile browsers that you need to keep in mind.
 
-1. On iOS and iPadOS, Safari is technically the only supported browser. Yes, you can install Chrome or Firefox, and you can even set them as a default, but they are not full browsers. They are still using the Safari rendering engine (WebKit). Therefore, for your web application to work for the Apple users, you have to assure support for WebKit and other technologies used in Safari.
-Important to remember, that since mobile browsers are not a one to one comparison to desktop, a project that works in desktop version of Safari might still need more adjustments to work properly on mobile version of the browser.
+<span id="apple-browsers">1. On iOS and iPadOS, Safari is technically the only supported browser. Yes, you can install Chrome or Firefox, and you can even set them as a default, but they are not full browsers. They are still using the Safari rendering engine (WebKit). Therefore, for your web application to work for the Apple users, you have to assure support for WebKit and other technologies used in Safari.
```

Comment:
Currently the placement of `span` breaks list formatting on the website (I checked using the [preview tool](https://www.theodinproject.com/lessons/preview)).

This should fix it:

```suggestion
1. <span id="apple-browsers">On iOS and iPadOS, Safari is technically the only supported browser. Yes, you can install Chrome or Firefox, and you can even set them as a default, but they are not full browsers. They are still using the Safari rendering engine (WebKit). Therefore, for your web application to work for the Apple users, you have to assure support for WebKit and other technologies used in Safari.
```